db/data/users.js

```diff
@@ -29,13 +29,18 @@ const userSchema = new mongoose.Schema({
     },
     email: {
       type: String,
+      required: [true, 'Please add a email'],
```

Comment:
We already have a validation message for this

Reply:
I didn't see that but it doesn't really hurt to leave it in.